### PR TITLE
feat: permissions variable type changed to set

### PR DESCRIPTION
**Pull request by ivasm-ko**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -121,14 +121,13 @@ variable "virtual_networks" {
...
-      object_id = null
-      role      = null
-    }
-  ]
+  type = set(object({
+    name      = string
+    object_id = string
+    role      = string
+  }))
+  description = "Storage permision set"
```
Comment (owlleg6) [inline variables.tf:129]: Set of objects, where you can assign role to certain principal and name this assignment.

Comment (ivasm-ko) [reply variables.tf]: updated

**Review by omartushevskyi — APPROVED** [review]
LGTM

**Review by owlleg6 — APPROVED** [review]
Lgtm

**Pull request merged**